### PR TITLE
Icepack in fesom2

### PR DESCRIPTION
**Pull request by lzampier**
Documentation regarding the Icepack implementation in FESOM2. More to follow over the next weeks/months, also in relation to users' input.

### CODE REVIEW

File: doc/icepack_in_fesom.rst [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+The Icepack environment is separated from the rest of FESOM2 and consists of a single big module with multiple submodules. Almost all the variables are private and are not visible by the FESOM2 code. The variables exchange between Icepack and FESOM2 takes place through the passing subroutines ``fesom_to_icepack`` and ``icepack_to_fesom``.
+
+Frequently asked questions
+==========================
+
+Should I use Icepack for my simulations?
+""""""""""""""""""""""""""""""""""""""""
+
+It depends on your scientific questions. Icepack might be a good option if you are interested to It depends on your scientific questions. Icepack might be a good option if you are interested in sea ice processes in polar regions. In principle, the employment of Icepack should not negatively affect the ocean state but could make FESOM2 slower.
```
Comment (koldunovn) [inline doc/icepack_in_fesom.rst]: Not complete sentence.

**Comment by koldunovn** [comment]
@lzampier Great that you did it! Apart from one small thing I have no issues with this PR. Can I merge?

File: doc/icepack_in_fesom.rst (hunk 1 above)
Comment (lzampier) [reply doc/icepack_in_fesom.rst]: Corrected! Thanks for spotting it :)

**Comment by lzampier** [comment]
> @lzampier Great that you did it! Apart from one small thing I have no issues with this PR. Can I merge?

Yes @koldunovn, you can go ahead! Thank you :)

**Pull request merged**